core/src/main/java/org/frankframework/jdbc/MessageStoreSender.java

```diff
@@ -84,6 +85,9 @@ public void configure() throws ConfigurationException {
 			paramList.configure();
 		}
 		setType(StorageType.MESSAGESTORAGE.getCode());
+		if (StringUtils.isBlank(getSlotId())) {
+			throw new ConfigurationException("'slotId' has to be configured");
+		}
```

Comment:
This was not a requirement before, but when `slotId` was blank, this used to result in invalid queries.
So I made this a requirement.